### PR TITLE
Update Spatial Service Connectivity CT

### LINKED ISSUE #215: Update of Spatial Service Connectivity Template
## Description & Tasks
IfcRelServicesBuildings has been deprecated and replace if IfcReferencedInSpatialStructure Concept template needs to be Updated.
- [x] Update mvdXML
- [x] Update Documentation

### Discussion Points

None at present

### Links
[Miro Element](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362317036395&cot=14)
[Miro Task Card](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362318411840&cot=14)

### Decisions 
TBC

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
Fixes #215 

update of template replacing  IfcRelServicesBuildings with the  new generic form IfcReferencedInSpatialStructure

### CODE REVIEW

File: IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md [hunk 1]
```diff
@@ -1 +1,3 @@
-A system, such as a distribution system, services a particular spatial structure, either a total building, a building section, or a storey, or any part of these structures.
+A system, such as a distribution system, services a particular spatial structure, either an entire facility (and it specific subtypes e.g. buildings, bridges, roads or marine facilities, a facility part, or a building storey, or any part of these structures.
```
Comment (pjanck) [inline IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md:1]:
```suggestion
A system, such as a distribution system, services a particular spatial structure, either an entire facility (and its specific subtypes e.g. buildings, bridges, roads, or marine facilities), a facility part, or a building storey, or any part of these structures.
```

Guessing the correct location of the closing parenthesis.

File: IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+The system in question should be human identifyable by its _Name_ attribute.
+The use of _IfcRelReferencedInSpatialStructure_ in this template provides the relationship across spatial and functional heirarchies allowing the spatial covererage of functional systems.
```
Comment (pjanck) [inline IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md:3]:
```suggestion
The use of _IfcRelReferencedInSpatialStructure_ in this template provides the relationship across spatial and functional hierarchies allowing the spatial coverage of functional systems.
```

File: IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md [hunk 3]
```diff
@@ -1 +1,3 @@
-A system, such as a distribution system, services a particular spatial structure, either a total building, a building section, or a storey, or any part of these structures.
+A system, such as a distribution system, services a particular spatial structure, either an entire facility (and it specific subtypes e.g. buildings, bridges, roads or marine facilities, a facility part, or a building storey, or any part of these structures.
+The system in question should be human identifyable by its _Name_ attribute.
```
Comment (pjanck) [inline IFC4x3/Templates/Object Connectivity/Spatial Service Connectivity/Documentation.md:2]:
```suggestion
The system (_IfcSystem_) in question should be human identifiable by its _Name_ attribute.
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Typos :)

**Review by pjanck — APPROVED** [review]
lftm

**Comment by AlexBrad1eyCT** [comment]
@SergejMuhic, @larswik or @TLiebich a secondary check and approval would be appreciated then this can be merged

**Review by larswik — APPROVED** [review]

**Comment by czapplitec** [comment]
There is an issue that we have a requirement to reference an IfcGroup by an IfcSpatialElement.
Proposal 1: generalize this CT, using IfcGroup instead of IfcSystem, but then this CT needs to be renamed.
Proposal 2: define Group Reference as another CT.
@evandroAlfieri

**Comment by larswik** [comment]
The way IfcSystem is defined: "composed for a common purpose or function or to provide a service" makes me think that it fits well with this template "Spatial Service Connectivity". IfcGroup as the general concept is defined as "an generalization of any arbitrary group". For that reason I would vote for Proposal 2 as a more generic template such as "Spatial Group Connectivity".

**Comment by evandroAlfieri** [comment]
Indeed, since we made possible to reference IfcGroup in SS I think this should be reflected in some CT.
Also, do not forget some "constraints" we noted while testing System topic + working on storylines. See the first 3 here: https://github.com/IFCRail/IFC-Repository-Sandbox/blob/master/Schema%20constraints/Constraints.md

**Pull request merged**